src/bot/slack_bot.py

```diff
@@ -50,7 +50,7 @@ def start_timer(client, say, event):
     time.sleep(TIMER_DURATION)
     print(f"{TIMER_DURATION} second timer complete!")
 
+    timer_active = False
```

Comment:
i've moved this here because currently if you react while the reacts are being populated, it doesn't count